native/src/components/EmbeddedOffers.tsx

```diff
@@ -27,13 +36,15 @@ const EmbeddedOffers = ({ category, cityCode, languageCode, goBack }: EmbeddedOf
         cityContentPath: category.path,
       })
       return (
-        <MalteHelpFormOffer
-          categoryPageTitle={category.title}
-          url={url}
-          malteHelpFormOffer={offer}
-          cityCode={cityCode}
-          onSubmit={goBack}
-        />
+        <Container withMargin={!!category.content}>
```

Comment:
I think we should not make this necessary on every switch-case element, but move it to a higher level. Especially when adding more cases for other offers this is a lot of duplication and makes the code harder to read and maintain.

Suggestion:
```

const EmbeddedOffer = (embeddedOfferProps: EmbeddedOfferProps): ReactElement | null => {
  const { category, cityCode, languageCode, goBack } = embeddedOfferProps
  const offer = category.embeddedOffers[0]
  switch (offer?.alias) {
    case SPRUNGBRETT_OFFER_ALIAS:
      return (
        <Container withMargin={!!category.content}>
          <SprungbrettOffer sprungbrettOffer={offer} languageCode={languageCode} />
        </Container>
      )
    case MALTE_HELP_FORM_OFFER_ROUTE: {
      const url = urlFromRouteInformation({
        route: CATEGORIES_ROUTE,
        languageCode,
        cityCode,
        cityContentPath: category.path,
      })
      return (
        <Container withMargin={!!category.content}>
          <MalteHelpFormOffer
            categoryPageTitle={category.title}
            url={url}
            malteHelpFormOffer={offer}
            cityCode={cityCode}
            onSubmit={goBack}
          />
        </Container>
      )
    }
    default:
      return null
  }
}

const EmbeddedOffers = (embeddedOfferProps: EmbeddedOfferProps): ReactElement | null => {
  const { category } = embeddedOfferProps
  return (
    <Container withMargin={!!category.content}>
      <EmbeddedOffer {...embeddedOfferProps} />
    </Container>
  )
}
```

Reply:
Good point, thanks!